crates/contract/tests/sandbox/common.rs

```diff
@@ -74,7 +74,7 @@ const DUMMY_MIGRATION_CONTRACT_PACKAGE_NAME: &str = "test-migration-contract";
 /// nodes usually attach the maximum available gas.  
 /// For testing, we use this constant to attach a fixed amount to each call and detect if gas usage increases  
 /// unexpectedly in the future.
-pub const GAS_FOR_VOTE_RESHARED: Gas = Gas::from_tgas(22);
+pub const GAS_FOR_VOTE_RESHARED: Gas = Gas::from_tgas(34);
```

Comment:
was this needed as part of the gas experiments only? Because now it looks like a big bump, please revert if not needed